main.py

```diff
@@ -0,0 +1,119 @@
+from fastapi import FastAPI, APIRouter
+from fastapi.middleware.cors import CORSMiddleware
+
+from models.account_model import Account
+
+app = FastAPI(
+    title="GenZ Co-Pilot Plugin API",
+    description="Allows co-pilot to fetch ",
+    version="1.0.0",
+    servers=[
+        {
+            "url": "http://localhost:8000",
+            "description": "GenZ Co-pilot Plugin API that can response with lead account information"
+        },
+    ]
+)
+
+# Add the CORS middleware to the application
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=["*"],  # List of allowed origins
```

Comment:
The CORS middleware is currently set to allow all origins, methods, and headers. Are there specific security considerations or use cases that led to this decision?